wapi/curves.py

```diff
@@ -75,6 +75,73 @@ def access(self):
 class TimeSeriesCurve(BaseCurve):
     def get_data(self, data_from=None, data_to=None, time_zone=None, filter=None,
                  function=None, frequency=None, output_time_zone=None):
+        """ Getting data from Time Series curves
+                
+        A Time Series curves holds a single time series. 
+        This is used for actual values, backcasts, normals, etc. This function
+        fetches data between two given timestamps. It also possible to process
+        the curve directly in the API using filter and aggregation functions. 
+        This can be used with great effect to reduce the amount of data 
+        retrieved if the full set of details is not needed. 
+        All time series are returned in a :class:`wapi.util.TS` object.
+
+        Parameters
+        ----------
+        
+        data_from: time-stamp, optinal
+            start date (and time) of data to be fetched. If not given, the start
+            date of the returned timeseries will be the first date with data
+            available. If only the date (without time) is 
+            given, the time is assumed to be 00:00. The timestamp can be 
+            provided in any of the following types (also valid without time):
+            
+            * datestring in format '%Y-%M-%DT%h:%m:%sZ',
+              eg '2017-01-01' or '2018-12-16T13:45:00Z'
+            * pandas.Timestamp object
+            * datetime.datetime object
+            
+        data_to: time-stamp, optional
+            end date (and time) of data to be fetched. The time-stamp can be 
+            provided in the same types as ´data_from´. 
```

Comment:
Same problem as before. The ´ character fails building for python 2.7 for both PRs.